lms/templates/course_modes/choose.html

```diff
@@ -86,19 +86,33 @@ <h3 class="title">
                                 <div class="wrapper-copy">
                                     <span class="deco-ribbon"></span>
                                     % if has_credit_upsell:
-                                    <h4 class="title">${_("Pursue Academic Credit with a Verified Certificate")}</h4>
+                                    % if content_gating_enabled:
```

Comment:
I assumed that since the majority of this work was going into the open source repo, that we want to make it available to open source users. Will verify with team.